src/Utopia/Messaging/Adapters/SMS/ClickSend.php

```diff
@@ -41,6 +41,15 @@ protected function process(SMS $message): string
             fn ($to) => \ltrim($to, '+'),
             $message->getTo()
         );
+        $body = array();
```

Comment:
Let's use modern array syntax here

```suggestion
        $body = [];
```